presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -581,6 +582,11 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         CACHE_ENABLED,
                         "Enable cache for hive",
                         cacheConfig.isCachingEnabled(),
+                        false),
+                booleanProperty(
+                        EXECUTION_BASED_MEMORY_ACCOUNTING_ENABLED,
+                        "Enable execution based memory accounting for ORC writers",
```

Comment:
Not sure it makes sense to mention ORC writers. This is just the first memory accounting fix we are putting behind this flag.
```suggestion
                        "Enable loose memory accounting to avoid OOMing existing queries",
```